integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -211,7 +254,7 @@ def mongo_doc_to_haystack_doc(self, mongo_doc: Dict[str, Any]) -> Document:
         Converts the dictionary coming out of MongoDB into a Haystack document
```

Comment:
I'd make `mongo_doc_to_haystack_doc` private as it's not used outside this module